assets/js/components/surveys/CurrentSurvey.test.js

```diff
@@ -80,6 +80,158 @@ describe( 'CurrentSurvey', () => {
 		expect( container ).toMatchSnapshot();
 	} );
 
+	it( "should render a multi select question when the `question_type` is 'multi_select'", async () => {
```

Comment:
[I saw you mentioned breaking these up like for #3762](https://github.com/google/site-kit-wp/issues/3762#issuecomment-895417521) 👍🏻

Reply:
Broken up now! :+1: